content/next/index.md

```diff
@@ -53,19 +53,12 @@ BREAKING CHANGE: `extends` key in config file is now used for extending other co
 
 ### Commit message with `!` to draw attention to breaking change
 ```
-feat!: drop support for Node 6
+feat!: enforce authentication for all requests
```

Comment:
What about a more product oriented example?

`feat!: send an email to the customer when the product is shipped`

Reply:
cool, I like your example. I updated to `send an email to the customer when a product is shipped` (I felt free to replace "the" through "a").

Reply:
(and of course thanks a lot for your review, your time and energy for this whole project! ❤️ )